_guides/deploying-with-ksonnet.md

```diff
@@ -0,0 +1,519 @@
+---
+title: Deploying with KSonnet
+collection: How-to Guides
+layout: article
+section: On-prem
+category: component
+order: 1
+since: 20190119
+---
+
+One of the first problems this article solves is simplifying how users configure the applications they deploy to Kubernetes clusters.
```

Comment:
This article is about deploying elastic.io platform to Kubernetes, not any application.

Reply:
Sure.